lib/ash/resource/validation/action_is.ex

```diff
@@ -4,14 +4,16 @@ defmodule Ash.Resource.Validation.ActionIs do
 
   @impl true
   def validate(changeset, opts, _context) do
-    if changeset.action.name == opts[:action] do
+    opts = Keyword.put(opts, :action, List.wrap(opts[:action]))
+
+    if changeset.action.name in opts[:action] do
       :ok
     else
       # We use "unknown" here because it doesn't make sense to surface
       # this error to clients potentially (and this should really only be used as a condition anyway)
-      [message: message, vars: vars] = describe(opts)
+      [message: message] = describe(opts)
```

Comment:
lets not pattern match on `describe/1` and instead do `description = describe(opts)` and then `.exception(error: description[:message], vars: opts[:vars])`